telemetry.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+package openmind;
+
+option csharp_namespace = "OpenMind";
+
+service TelemetryService {
+  /**
+   * TODO (BNR): We need endpoints for:
+   *             1. Version number - should respond with the server version
+   *             2. Supported devices - should respond with "Medtronic Summit RC+S"
+   *             3. Inspect Repository - should respond with the cached bridges/devices
+   */
+
+  /**
+   * List the version number for the Summit Server
+   */
+  rpc VersionNumber (VersionNumberRequest) returns (VersionNumberResponse); 
+  /**
+  * List supported devices.
+  */
+  rpc SupportedDevices (SupportedDevicesRequest) returns (SupportedDevicesResponse);
+  /**
+   * Inspect Repository.
+   */
+  rpc InspectRepository (InspectRepositoryRequest) returns (InspectRepositoryResponse);
+}
+
+message VersionNumberRequest {}
+
+message VersionNumberResponse {
+  
+  /**
+   String with the version number for the Summit Server 
+   */
+  string version_number = 1;
+}
+
+message SupportedDevicesRequest {}
+
+message SupportedDevicesResponse {
+  /**
+   * List with supported devices.
+   */
+  repeated string supported_devices = 1;
+
+}
+
+message InspectRepositoryRequest{
+}
+
+message InspectRepositoryResponse{
+
+  /**
+   * repeated cached bridges
+   */
+  repeated CachedBridges cached_bridges = 2;
+
+  /**
+   * repeated cached devices
+   */
+  repeated CachedDevices cached_devices = 3;
+}
+
+message CachedBridges{
+  /**
+   * bridge
+   */
+  string bridge = 1;  
+}
+
+message CachedDevices{
+  /**
+   * device
+   */
+  string device = 1;  
+}
```

Comment:
any reason why we're not using a repeated string instead of a repeated `CachedDevices`? I'd recommend something like this:

```message InspectRepositoryResponse{

  /**
   * repeated cached bridges
   */
  repeated string cached_bridges = 2;

  /**
   * repeated cached devices
   */
  repeated string cached_devices = 3;
}
```

I guess the benefit of having the `CachedDevices` is that we can have multiple different fields. If we go that route the name should be singular, `CachedDevice`.